pkgs/by-name/ai/aider/package.nix

```diff
@@ -0,0 +1,97 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+  git,
+  portaudio,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "aider";
+  version = "0.47.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "paul-gauthier";
+    repo = "aider";
+    rev = "v${version}";
+    hash = "sha256-kDGmP6X59H0nGPChSTiOBTbnJS3NRQqywjpo9qoB/rg=";
+  };
+
+  build-system = with python3.pkgs; [
+    pythonRelaxDepsHook
```

Comment:
Does this belong here or in nativeBuildInputs?

Reply:
Good question. I think I put it there as per @eclairevoyant's request.

Reply:
https://nixos.org/manual/nixpkgs/stable/#buildpythonpackage-parameters describes them as:
```
nativeBuildInputs ? []: Build-time only dependencies. Typically executables.

build-system ? []: Build-time only Python dependencies. Items listed in build-system.requires/setup_requires.
```
so `nativeBuildInputs` is more reasonable

Reply:
sure let's move it then.